hera_pspec/uvpspec.py

```diff
@@ -26,7 +26,14 @@ def __init__(self):
         by hera_pspec.
         """
         # Summary attributes
-        self._Ntimes = PSpecParam("Ntimes", description="Number of unique times.", expected_type=int)
+        # Note that in the past (pre-v0.5) Ntimes was the number of unique times in 
+        # either of the underlying visibilitydatasets (i.e. time_1_array and time_2_array),
```

Comment:
Missing space

Reply:
I'd rather say "the number of unique times in **a concatenation** of the underlying visibility datasets"

Reply:
I've gone with "union" since we're talking about sets and uniqueness here :-)